code/src/components/DropDown.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import "./dropdown.css";
+
+// const dropDownOptionsArray = ["-", "0-10 km", "20-50 km", "50+ km"];
+
+const DropDown = ({ dropdownText, dropdownAnswer, onDistanceChange }) => {
+
+  return (
+    <section className="section-container">
+      <div className="form-container">
+        <label tabIndex="0" htmlFor="drop-down"><h2>{dropdownText}</h2>
+          <select
+            name="distances" //not sure if necessary
+            onChange={(event) => onDistanceChange(event.target.value)}
+            value={dropdownAnswer}
+            required
+            id="drop-down"
+          >
+            <option value="">-</option>
+            <option tabIndex="0" value="0-10 km">0-10 km</option>
+            <option tabIndex="0" value="20-50 km">20-50 km</option>
+            <option tabIndex="0" value="50+ km">50+ km</option>
+
+            {/* why "required" attribute doesn't work with <select> when I use map()? */}
```

Comment:
I think it does, the issue might be connected to the fact that you initialise `distance` state property in `<Form />` component as empty object `{}`, but later on if you select any value, it will be string